spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-r2dbc-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.cloud.gcp.sql.database-name=[database-name]
+spring.cloud.gcp.sql.instance-connection-name=[instance-connection-name]
```

Comment:
Could you add a comment that instance-connection-name needs to be in the format `[GCP_PROJECT_ID]:[INSTANCE_REGION]:[INSTANCE_NAME]`? I put just the instance, and got "Cloud SQL connection name is invalid, expected string in the form of "<PROJECT_ID>:<REGION_ID>:<INSTANCE_ID>", which I did not connect with the property until I saw the integration test.

Reply:
Good idea! This will help make the samples clearer, thank you.